### DISCUSSION TITLE
Distinguish between Available and "Not Yet Released" sites?

### DISCUSSION THREAD

**Discussion by smilinggoat** [juftin/camply#85] [Ideas]
I'm checking out camply for the first time and it's awesome! You've done some really impressive work! Thank you for your hard work!

One thing I've noticed when searching for sites is if a site is "Not Yet Released," camply returns returns a result indistinguishable from a site being Available.

For example, `camply campsites --campground 10044761 --start-date 2022-09-01 --end-date 2022-09-02` makes it look as if the campsites in that campground are available for those dates when (at the time of writing) they are listed as "Not Yet Released," [on the website](https://www.recreation.gov/camping/campsites/10124770).

Would it be possible to distinguish between the Available and Not Yet Released to get a notification for when a campground becomes available for booking?

Apologies if this has been previously addressed, I searched through the documentation & discussions but didn't see anything relevant.

Thanks!

**Comment by juftin** [comment]
Hey @smilinggoat, are you using the latest version of camply? I fixed something similar to this very recently https://github.com/juftin/camply/issues/83

Let me know if that doesn't fix it for you.

**Comment by smilinggoat** [comment]
Oh awesome, that worked! Thank you for your quick reply!

pip had installed version 0.3.0 so I had to force the upgrade with `pip install camply==0.3.1` but that did the trick.
